indico/modules/search/schemas.py

```diff
@@ -58,7 +58,7 @@ class Meta:
 
     event_id = fields.Int(attribute='id')
     type = fields.Constant(SearchTarget.event.name)
-    type_format = fields.String(attribute='type.name')
+    type_format = fields.String(attribute='type')
```

Comment:
⚠️ I think this breaks the export!

Reply:
Indeed, I trusted the property value, the latest one should fix it.